ffmpeg-convert-audio/functions/index.js

```diff
@@ -23,6 +23,27 @@ const fs = require('fs');
 const ffmpeg = require('fluent-ffmpeg');
 const ffmpeg_static = require('ffmpeg-static');
 
+/**
+ * Utility method to convert audio to mono channel using FFMPEG.
+*/
+function reencodeAsync(tempFilePath, targetTempFilePath) {
+  return new Promise((resolve, reject) => {
+    const command = ffmpeg(tempFilePath)
+      .setFfmpegPath(ffmpeg_static.path)
```

Comment:
Add 2 more spaces since this is line-wrapping. See example in Google style guide: https://google.github.io/styleguide/jsguide.html#formatting-function-expressions